src/app/app.component.html

```diff
@@ -1,5 +1,5 @@
 <h1>ExUI Common Lib Test Page</h1>
-<xuilib-service-messages></xuilib-service-messages>
+<xuilib-service-message [key]="test-key" [message]="'This is a test xuilib-service-message for the ExUI Common Lib Test Page. For demo purposes it can not be hidden.'"></xuilib-service-message>
```

Comment:
What is `[key]` binding to? `test-key` isn't a valid variable name so unlikely to be in `app.component.ts`...

Reply:
It's used to show / hide messages. When a message is hidden the key is added to the session cookie in order for the message to remain hidden for the duration of the session

Reply:
So what is `test-key`?

Reply:
It's the name I gave the test-key, nothing more! It's for the test page in the lib.

Reply:
Agree with @paul-graham  tast-key is not ideal variable name. But looks like its not part this PRs change?